src/test/java/spoon/test/position/PositionTest.java

```diff
@@ -1458,4 +1461,105 @@ public void testLambdaParameterPosition1() {
 				)
 		);
 	}
+
+	@ModelTest("src/test/java/spoon/test/position/testclasses/ExecutableReferencePositionTestClass.java")
+	void testExecutableReferencesInInvocation(Factory factory) {
+		CtType<?> type = factory.Type().getAll().get(0);
+		CtMethod<?> method = type.getMethodsByName("entrypoint").get(0);
+
+		CtInvocation<?> untypedInvocation = method.getBody().getStatement(2);
+		CtInvocation<?> typedInvocation = method.getBody().getStatement(3);
```

Comment:
Maybe add the statement you want as a comment above. Otherwise, if there are ever any changes to the indexing the person debugging this is doomed. Also for the other tests.

Reply:
I am not sure, the method contains the name of the method it analyses and the asserts below that contain the exact code already. Nevertheless, I added some comments :)